lib/errors.ts

```diff
@@ -27,6 +27,40 @@ export class BalenaError extends TypedError {
 BalenaError.prototype.code = 'BalenaError';
 BalenaError.prototype.exitCode = 1;
 
+/**
+ * @summary Balena too many requests
+ * @class
+ * @public
+ *
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.BalenaTooManyRequests()
+ */
+export class BalenaTooManyRequests extends BalenaError {
+	constructor() {
+		super('Too Many Requests');
+	}
+}
+BalenaTooManyRequests.prototype.code = 'BalenaTooManyRequests';
+
+/**
+ * @summary Balena invalid login credentials
+ * @class
+ * @public
+ *
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.BalenaInvalidLoginCredentials()
+ */
+export class BalenaInvalidLoginCredentials extends BalenaError {
+	constructor() {
+		super('Invalid email / password');
```

Comment:
How about 
```suggestion
		super('Invalid login credentials');
```
? Since users can also login with their username.